Packs/MicrosoftADFS/ReleaseNotes/1_0_8.md

```diff
@@ -0,0 +1,3 @@
+#### Modeling Rules
+##### Microsoft ADFS Collection
+Added a field to the fields' schema.
```

Comment:
the user doesn't know what is the schema, just change it to fix an issue with the modeling rule.